docs/troubleshooting_and_faq.md

```diff
@@ -36,12 +36,28 @@ running `mp-build`.
 
 Docker CLI recommends using a token when using docker hub instead of your login password, so users should authenticate their Docker Hub account before running `mp-build`.
 
-1. Generate a [dockerhub token](https://hub.docker.com/settings/security?generateToken=true).
-2. Open the `config.json` file in the project's root directory, then replace the key-value of `docker_registry` with the tag of
-   the Docker Hub repository prepended with `docker.io`. For example, if the docker tag is `username/docker-registry-test:tagname`
-   then the key-value will be `docker.io/username/docker-registry-test`.
+1. Create a docker hub container registry with the same name as the docker string in the `adapter_kinds`  key inside the `manifest.txt`  file.
```

Comment:
* I don't think we need or want to provide step-by-step instructions for this, but it might be helpful to prefix with something like "Go to hub.docker.com and [...]"
* I don't think the registry has to match the adapter kind name, does it? As long as it's the same name we use in step 3. If that's the case, it might be better to only recommend using the adapter kind name

Reply:
Sure, but you're very specific about using a particular key in a particular file - in addition, I think there's slightly different requirements for an adapter key vs a registry name. For example, this is a valid adapter key: `MyVeryCoolAdapterKey`, but registries have to be lowercase, and `myverycooladapterkey` is much harder to read - a user might want to change to something like `my_very_cool_adapter_key`, etc